x-pack/plugins/security_solution/common/api/asset_criticality/common.schema.yaml

```diff
@@ -0,0 +1,61 @@
+openapi: 3.0.0
+info:
+  title: Asset Criticality Common Schema
+  description: Common schema for asset criticality
+  version: 1.0.0
+paths: { }
+components:
+  parameters:
```

Comment:
We could have had these URL parameters not query paramters, e.g currently we have

`GET /internal/risk_score/criticality?id_field=host.name&id_value=host-1`

Where both params are required, would this be better? 

`GET /internal/risk_score/criticality/host.name/host-1

Reply:
Besides being order-independent, I think the query parameters give us a little more flexibility moving forward as we're able to add more parameters in a backwards-compatible way.